FirebaseInAppMessaging/Swift/Source/SwiftUIPreviewHelpers.swift

```diff
@@ -73,6 +74,30 @@ public struct InAppMessagingPreviewHelpers {
     )
   }
 
+  public static func defaultBackgroundColor() -> UIColor {
+    #if os(tvOS)
+      return UIColor.white
```

Comment:
I think there should be an additional check within these helpers that returns a light or dark color based on [interface style](https://developer.apple.com/documentation/uikit/uitraitcollection/1651063-userinterfacestyle).

Reply:
Makes sense, would you do a follow up PR to update?

Reply:
Added to #8416 description